runtime/src/proofs.rs

```diff
@@ -0,0 +1,48 @@
+use rstd::vec::Vec;
+
+#[derive(Debug)]
+pub struct Proof {
+    hash: Vec<u8>,
```

Comment:
may be makes sense to use the in built `Hash` type here or make this fixed array

Reply:
You mean the runtime hash ?